aiohasupervisor/models/backups.py

```diff
@@ -141,7 +141,7 @@ class FullBackupOptions(Request):
     name: str | None = None
     password: str | None = None
     compressed: bool | None = None
-    location: list[str | None] | str | None = DEFAULT  # type: ignore[assignment]
+    location: str | list[str | None] | None = DEFAULT  # type: ignore[assignment]
```

Comment:
Do we need a test also for this?

Reply:
Its in there. I added a test for each type option for `location`